src/BlockchainTestsFiller/ValidBlocks/bcStateTests/testOpcodesFiller.yml

```diff
@@ -11285,7 +11285,7 @@ Test_Opcode_5c:
       balance: 0x0100000000000000
       nonce: 0
       storage: {}
```

Comment:
Ori, was it a script generated test?

Reply:
Yes. There's a `testOpcodesGen.js` file in that directory that generates it.